platform/ios/src/MGLMapView.mm

```diff
@@ -1298,9 +1296,8 @@ - (void)handlePinchGesture:(UIPinchGestureRecognizer *)pinch
     else if (pinch.state == UIGestureRecognizerStateChanged)
     {
         CGFloat newScale = self.scale * pinch.scale;
-        double zoom = log2(newScale);
-        if (zoom < _mbglMap->getMinZoom()) return;
-        
+        double zoom = MAX(log2(newScale), _mbglMap->getMinZoom());
```

Comment:
It might even be unnecessary to compare against `getMinZoom()`, because `mbgl::TransformState` already enforces the minimum zoom level. (See also #3842.)